src/renderGraphiQL.js

```diff
@@ -19,9 +19,9 @@ export type GraphiQLOptions = {|
 |};
 
 // Ensures string values are safe to be used within a <script> tag.
-function safeSerialize(data) {
+function safeSerialize(data: mixed): string {
```

Comment:
Actually Flow is correct and you don't need `any` if you change the argument to `data: string | null`

Reply:
Oops, I thought we were passing other types here, not just strings. I ended up using `?string` here to match the types of the fields in `GraphiQLData`.